src/components/CarouselDots.js

```diff
@@ -67,8 +68,9 @@ class CarouselDots extends Component {
   }
 
   render() {
+    const { className = '', rtl } = this.props;
```

Comment:
Default value is not needed here.

Reply:
Changed in PR #403

Reply:
@Lukasz-pluszczewski this is fixed. Please review again :)